Content.Server/Destructible/Thresholds/Behaviors/SpawnEntitiesBehavior.cs

```diff
@@ -42,7 +47,17 @@ public void Execute(EntityUid owner, DestructibleSystem system, EntityUid? cause
                 {
                     for (var i = 0; i < count; i++)
                     {
-                        system.EntityManager.SpawnEntity(entityId, position.Offset(getRandomVector()));
+                        var spawned = system.EntityManager.SpawnEntity(entityId, position.Offset(getRandomVector()));
+
+                        if (!TransferForensics || !system.EntityManager.TryGetComponent<ForensicsComponent>(owner, out var forensicsComponent))
```

Comment:
any reason why this should only be on entities that don't have stack component?

Reply:
nope, just me being inattentive